mysql/Dockerfile.mariadb

```
@@ -1,5 +1,5 @@
 FROM mariadb
 
-COPY docker-healthcheck /usr/local/bin/
+ENV MARIADB_MYSQL_LOCALHOST_USER=1
 
-HEALTHCHECK CMD ["docker-healthcheck"]
+HEALTHCHECK CMD ["/usr/local/bin/healthcheck.sh", "--su=mysql", "--connect", "--innodb_initialized"]
```

One more minor nit -- I think we should let `PATH` make this easier to read: :smile:

```suggestion
HEALTHCHECK CMD ["healthcheck.sh", "--su=mysql", "--connect", "--innodb_initialized"]
```